includes/validation/class-amp-validated-url-post-type.php

```diff
@@ -1673,7 +1670,10 @@ public static function handle_validation_error_status_update() {
 				continue;
 			}
 			$term_group = AMP_Validation_Error_Taxonomy::sanitize_term_status( $status );
-			if ( null !== $term_group && $term_group !== $term->term_group ) {
+
+			$status_changed = ( $term->term_group | AMP_Validation_Error_Taxonomy::ACKNOWLEDGED_VALIDATION_ERROR_BIT_MASK ) !== $term_group;
```

Comment:
Doing a bit-wise OR here gives the acknowledged status code, and would then allow for comparison against the submitted status code (which is always acknowledged).